unit-tests.gs

```diff
@@ -81,3 +81,36 @@ function testSemVer () {
     assert(v2.compare(v1) === (-comp.result), 'Comparison between ' + comp.v2 + ' and ' + comp.v1 + ' did not return the expected value of ' + (-comp.result));
   });
 }
+
+/**
+ * Test all events from selected period. It won't send actual e-mails to you, but put content of them into log.
+ *
+ * !!! Execution of this function very ofted exceed maximum time (30s).
+ *
+ * @param {Date} testDate - First date to start test; by default 1 January of current year.
+ * @param {int} numberOfDaysToTest - Number of days to test; by default 365 days (1 year).
+ * @param {bool} printHTML - Whether or not print HTML mailContent into log; by default not.
+ */
+function testSelectedPeriod (testDate, numberOfDaysToTest, printHTML) {
+
+  testDate = testDate || new Date(new Date().getFullYear(), 0, 1, 6, 0 ,0);
```

Comment:
There should be no space before each comma.
A space is required after each comma.